valkey.conf

```diff
@@ -1672,8 +1672,10 @@ aof-timestamp-enabled no
 # cluster-migration-barrier 1
 
 # Turning off this option allows to use less automatic cluster configuration.
-# It both disables migration to orphaned masters and migration from masters
-# that became empty.
+# It disables migration of replicas to orphaned masters. Masters that become
+# empty due to losing their last slots to another master will not automatically
+# replicate from the master that took over their last slots. Instead, they will
+# remain as empty masters without any slots.
```

Comment:
For the release notes I suppose. Mention in PR description.

Reply:
yeah, the "yes" behavior is the same but the "no" behavior is now deterministic that the empty primary won't automatically join the shard that takes away its last slot.